installation/scripts/kyma-scripts/tiller-tls.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
We don't need to retry getting these credentials, as in your script it will always happen after Kyma is installed - so please remove it